src/main/daml/Daml/Finance/Interface/Account/Account.daml

```diff
@@ -22,13 +23,24 @@ type R = Reference
 -- | Type synonym for `View`.
 type V = View
 
--- | Controllers of the account (related to transfers).
+-- | Frozen details.
+data Frozen = Frozen
```

Comment:
no strong opinion, but I would rename the data type to something like `AccountFreezeDetails`

Reply:
I will change to `FreezeDetails`

Reply:
I did not rename the view field `frozen : Optional FreezeDetails`, the optional indicates if frozen or not